### DISCUSSION TITLE
Layout modes issue

### DISCUSSION THREAD

**Discussion by Shareni** [i3/i3#4553] [Q&A] [answered]
I've switched from Arco to Arch, and the layouts are not working as expected. I'm not sure of the proper i3 terminology, so i'll call every open program a window.

In Arco i could set multiple windows to be stacked, and move windows between them. In this mostly default i3, setting any window to stacking pulls all windows in that workspace into the first open window. Additional attempts at making the other windows stacked makes more borders, until i try to move the first window at which point everything goes back to tiling and the borders disappear. [Here's a video of what I mean](https://imgur.com/a/n93x37R) 

What do i do to make it behave as expected? I've seen in some guides that changing the tiling orientation makes a new container, but it's not working. A possible clue is that in Arco (i3-gaps i think) changing the orientation did nothing, while it works in the default.

**Comment by Airblader** [comment, marked as answer]
I don't think I really understand what you're describing, so I'll answer in general terms and hope it addresses your issue.

If you just want the old behavior back, the easiest thing to do would simply be to go to Arco and have a look at the i3 configuration that it ships. Everything is in there one way or another.

As for stacking windows, you can create a horizontal / vertical split and only then set the layout to stacked. Say you have two windows open (`H[W1 W2]`), and focus is on the last one (`W2`). Then `split vertical, layout stacked` results in `H[W1 S[W2]]` instead of `S[W1 W2]` which would happen if you just change the layout.

> in Arco (i3-gaps i think) 

i3-gaps doesn't behave differently from i3 here.

**Comment by Shareni** [comment]
Thank you. I've read those instructions before, but couldn't get them to work before reading yours. 

I'm guessing it's got something to do with autotiling in Arco since it swaps between h and v for every new window.
